README.md

```diff
@@ -0,0 +1,56 @@
+# YOUTUBE HOMEPAGE
+
+> This is a project aimed at implementing lessons learned on HTML5 AND CSS3. The project provides a simple layout of the Youtube homepage and affords me the opportunity to learn and implement what i learnt.
+
+![screenshot](src/assets/images/screenshot.png)
+
```

Comment:
Your screenshot image URL is not correct. You did not change the one you copied and that is the reason why the image is not rendering. Kindly fix this.